src/code42cli/cmds/departing_employee.py

```diff
@@ -102,26 +100,21 @@ def bulk(state):
 )
 @read_csv_arg(headers=DEPARTING_EMPLOYEE_CSV_HEADERS)
 @sdk_options()
-@click.pass_context
-def bulk_add(ctx, state, csv_rows):
+def bulk_add(state, csv_rows):
+    sdk = state.sdk  # Force initialization of py42 to only happen once.
+
     def handle_row(username, cloud_alias, departure_date, notes):
         if departure_date:
             try:
                 departure_date = click.DateTime(formats=[DATE_FORMAT]).convert(
                     departure_date, None, None
                 )
             except click.exceptions.BadParameter:
-                message = "Invalid date {}, valid date format {}".format(
-                    departure_date, DATE_FORMAT
+                message = (
+                    f"Invalid date {departure_date}, valid date format {DATE_FORMAT}."
                 )
                 raise Code42CLIError(message)
-        ctx.invoke(
-            add,
-            username=username,
-            cloud_alias=cloud_alias,
-            departure_date=departure_date,
-            notes=notes,
-        )
+        _add_departing_employee(sdk, username, cloud_alias, departure_date, notes)
```

Comment:
`ctx.invoke()` was the culprit here but I still like this better. I don't know what `ctx.invoke()` does behind the scenes and it seems like unnecessary to handle off the execution back to the context